intern/remote/boss/remote.py

```diff
@@ -850,6 +858,22 @@ def delete_metadata(self, resource, keys):
         self.metadata_service.set_auth(self._token_metadata)
         self.metadata_service.delete(resource, keys)
 
+    def parse_bossURI(self, uri: str) -> Resource:
```

Comment:
I've been meaning to start using types. Are type hints done in this manner Python 2 compatible?

Reply:
I updated to an annotation that I know is compatible with all Python versions (comment-form) just to be sure :) I think that 2.7 is supported by mypy but it may depend on the environment. Should be fixed now!